zellij-tile/src/lib.rs

```diff
@@ -39,3 +39,19 @@ macro_rules! register_plugin {
         }
     };
 }
+
```

Comment:
Would these macros be able to live in the status bar?

Reply:
These are used in tab-bar too, and anywhere pretty much where you want to color stuff, those would be needed, so I figured maybe here is the best place for them?

Reply:
I think that's fair, but I don't quite think they are *essential* to the functioning of the API (especially if we are going to have "invisible" plugins soon that don't need a `Pane` to run).

How would you feel about spinning out a `zellij-tile-extra` library or something like that in the main repo (just like `zellij-tile`) that can hold non-critical but potentially quite useful helper functions / macros? Like I said, I'm quite invested in keeping `zellij-tile` as slim and minimal as possible, but I'd be happy putting almost anything in a library like `zellij-tile-extra`. I think having that additional "utility" library could actually help us clean up the plugins we have so far (eventually) :slightly_smiling_face: